src/edu/csus/ecs/pc2/clics/API202306/ContestService.java

```diff
@@ -389,9 +388,15 @@ private Response HandleContestCountdownPauseTime(SecurityContext sc, String cont
         // MIN_VALUE is returned on format error
         if(pauseTime != Long.MIN_VALUE) {
             // want to stop the countdown with this many milliseconds left
-            // TODO: tell PC2 to stop countdown when clock is 'pauseTime' ms away from start
-            controller.getLog().log(Log.WARNING, LOG_PREFIX + contestId + ": countdown_pause_time not implemented");
-            return Response.status(Status.NOT_MODIFIED).entity("Unable to set countdown_pause_time to " + pauseTime + " ms").build();
+            // tell PC2 to stop countdown when clock is 'pauseTime' ms away from start
+            boolean success = setScheduledStart(null, countdownPauseTime);
+            if (success) {
+                controller.getLog().log(Log.INFO, LOG_PREFIX + contestId + ": countdown paused (set contest start time to null)");
```

Comment:
I think this log message is a litle confusing: it says that the countdown is paused, and that the start time was set to null.  Those are both true -- but it would be clearer if it _also_ included the (formatted) "countdownPauseTime" -- so that the person reading the log understands that _setting the pause time to that value_ is _**also**_ a part of what has happened.